resources/providers/awslib/iam/user.go

```diff
@@ -18,36 +18,27 @@
 package iam
 
 import (
+	"bytes"
 	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
 	iamsdk "github.com/aws/aws-sdk-go-v2/service/iam"
 	"github.com/aws/aws-sdk-go-v2/service/iam/types"
+	smithy "github.com/aws/smithy-go"
 	"github.com/elastic/cloudbeat/resources/fetching"
 	"github.com/elastic/cloudbeat/resources/providers/awslib"
+	"github.com/gocarina/gocsv"
+	"github.com/pkg/errors"
 	"strings"
 	"time"
 )
 
-type User struct {
-	Name        string
-	AccessKeys  []AccessKey
-	MFADevices  []AuthDevice
-	LastAccess  time.Time
-	Arn         string
-	HasLoggedIn bool
-}
-
-type AuthDevice struct {
-	types.MFADevice
-	IsVirtual bool
-}
+const (
+	rootAccount = "<root_account>"
+	maxRetries  = 5
+	interval    = 3 * time.Second
+)
 
-type AccessKey struct {
-	AccessKeyId  string
-	Active       bool
-	CreationDate time.Time
-	LastAccess   time.Time
-	HasUsed      bool
-}
+var countRetries = 0
```

Comment:
When the counter will be zero again?
Can it be part of function closure instead of on the global stack?

Reply:
Sure, good catch!